### ISSUE TITLE
How do I add custom nunjucks filters to use in my templates?

### ISSUE THREAD

**Issue by dhruvdcoder** [mgmeyers/obsidian-zotero-integration#238]
Thanks for the excellent plugin. I wanted to create some custom formatting based on the content of my `annotation.comment`. The logic is a bit complicated and would be best implemented as a custom filter. How can I add my own numjucks custom filter definitions in the plugin? Thanks!

**Comment by DanielRunningen** [comment]
I tried adding a `<script>` tag at the top of a template and followed the [Nunjucks documentation](https://mozilla.github.io/nunjucks/api#custom-filters) to add a custom filter, but this doesn't seem to load in time for Nunjucks to implement it for the rest of the template.

**Comment by dcgk** [comment]
Yes, this would be excellent. I keep running into the limitations of Nunjucks. 
Or, is it possible to run javascript to do some preprocessing on the data?
Greetings and Thanks for this brilliant plugin!
